tests/Support/SupportStrTest.php

```diff
@@ -151,9 +151,11 @@ public function testStrContains()
         $this->assertTrue(Str::contains('taylor', 'taylor'));
         $this->assertTrue(Str::contains('taylor', ['ylo']));
         $this->assertTrue(Str::contains('taylor', ['xxx', 'ylo']));
+        $this->assertTrue(Str::contains(123, '1'));
```

Comment:
We don't officially want to allow this. Just strings or objects with __toString. Can you add a test with something like that, instead of this.

Reply:
Done, I used the `StringableObjectStub` class already in this test file.